ddp-workspace/projects/ddp-pancan/src/app/components/auth/auth.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component } from '@angular/core';
+import { SessionMementoService } from 'ddp-sdk';
+
+@Component({
+    selector: 'app-auth',
+    template: `
+        <ddp-language-selector></ddp-language-selector>
+        <div class="auth__login">
+            <ddp-sign-in-out></ddp-sign-in-out>
+        </div>
+        <a *ngIf="!isAuthenticated"
+           mat-flat-button
+           class="join-button btn-common"
+           routerLink="count-me-in"
```

Comment:
Would you use a constant from `app-routes.ts` there has to have a consistency?

Reply:
thanks! fixed